.github/workflows/ci-buildprevious.yml

```diff
@@ -26,9 +26,9 @@ jobs:
           echo "::add-path::$HOME/.idris2/bin"
       - name : Build previous version
         run: |
-          wget https://www.idris-lang.org/idris2-src/idris2-0.2.2.tgz
-          tar zxvf idris2-0.2.2.tgz
-          cd Idris2-0.2.2
+          wget https://www.idris-lang.org/idris2-src/idris2-0.3.0.tgz
+          tar zxvf idris2-0.3.0.tgz
+          cd Idris2-0.3.0
           make bootstrap-build && make install
```

Comment:
Could we make this version a variable in the `env:` section?
This will be more robust than changing the name in (at the moment) 3 different places.

Reply:
Changed, thanks for the suggestion. :3